api/tests/unit/application/certification-courses/index_test.js

```diff
@@ -113,6 +113,30 @@ describe('Unit | Application | Certifications Course | Route', function () {
       // then
       expect(response.statusCode).to.equal(200);
     });
+
+    it('return forbiden access if user has METIER role', async function () {
```

Comment:
```suggestion
    it('returns forbidden access if user has METIER role', async function () {
```

(je ne mets pas un commentaire pour chaque mais il y en a quelques autres)